src/utils/helpers/buildMapper.ts

```diff
@@ -40,51 +42,34 @@ const dateToTimezone = (date: string) => {
   });
 };
 
-export const buildMapper = (data: Build[], currentBuildId: number) => {
```

Comment:
I'm not a huge fan of creating these kinds of intermediate data objects (you might call the return value of `buildMapper` a "mapped build") that spread through the front-end codebase and make it harder to reason about the types being passed around. I realized I could fairly easily break this up into separate, smaller functions and just call those where necessary, rather than creating these "mapped builds" and passing them to other components.